### PR TITLE
Closes: #1133

### PR DESCRIPTION
**Pull request by ilariaventurini**
Repeated values in class names.
For more information, see issue #1133 .

### CODE REVIEW

File: plugins/prefixIds.js [hunk 1]
```diff
@@ -146,6 +146,10 @@ exports.fn = function(node, opts, extra) {
         if(prefix === false){
             return escapeIdentifierName(name);
         }
+        if (name.startsWith(escapeIdentifierName(prefix) + opts.delim)) {
```
Comment (XhmikosR) [inline plugins/prefixIds.js]: You should combine this check with the  previous one since they return the same value.

Comment (ilariaventurini) [reply plugins/prefixIds.js]: @GreLI @XhmikosR Thanks! Solved

**Comment by XhmikosR** [comment]
@ilariaventurini You have unrelated changes though.

**Comment by TrySound** [comment]
Prefixing with multipass option is fixed.

**Pull request closed without merging**